FATAL	Fatal error	multiple targets cannot be specified

**Describe the bug**
Our pipeline was working yesterday, and has been working for months. Now, our Trivy FS task is failing with the error:
FATAL	Fatal error	multiple targets cannot be specified

**Trivy Azure Pipeline Task Verison**
1.8.0

**Hosted or Self-hosted Agent**
Hosted agent

**Trivy as Docker?**
Docker form

**Error message**
```
Run requested using docker...
Using Trivy image: aquasec/trivy:latest
Configuring options for image scan...
/usr/local/bin/docker run --rm -v /Users/itadmin/.docker:/root/.docker -v /Users/itadmin/myagent/_work/_temp/:/tmp -v /Users/itadmin/myagent/_work/_temp/.cache:/root/.cache/trivy -v /var/run/docker.sock:/var/run/docker.sock -v /Users/itadmin/myagent/_work/7/s:/src --workdir /src aquasec/trivy:latest fs --exit-code 1 --format json --output /tmp/trivy-results-0.3767666145245814.json --severity HIGH,CRITICAL --ignorefile /src/pipeline/config/.trivyignore  --db-repository ghcr.io/aquasecurity/trivy-db:2,public.ecr.aws/aquasecurity/trivy-db:2 --java-db-repository ghcr.io/aquasecurity/trivy-java-db:1,public.ecr.aws/aquasecurity/trivy-java-db:1 --scanners vuln,misconfig,secret app
2025-02-20T18:15:33Z	FATAL	Fatal error	multiple targets cannot be specified
Scan local filesystem
```
```
##[error]Failed: Trivy detected problems.
##[error]Trivy seems to have failed so no output path to generate reports from.
```

--- a/trivy-task/index.ts
+++ b/trivy-task/index.ts
@@ -159,9 +159,6 @@ function configureScan(
   if (ignoreUnfixed) {
     runner.arg(['--ignore-unfixed']);
   }
-  if (options.length) {
-    runner.arg(options.split(' '));
-  }
 
   // if scanners haven't been set in the options, add them here
   if (
@@ -169,6 +166,10 @@ function configureScan(
     !options.includes('--security-checks')
   ) {
     runner.arg(['--scanners', scanners]);
+  }
+
+  if (options.length) {
+    runner.line(options);
   }
 
   runner.arg(target);

--- a/trivy-task/task.json
+++ b/trivy-task/task.json
@@ -8,7 +8,7 @@
   "helpUrl": "https://github.com/aquasecurity/trivy-azure-pipelines-task",
   "category": "Test",
   "author": "Aqua Security",
-  "version": { "Major":1,"Minor":11,"Patch":81 },
+  "version": { "Major":1,"Minor":11,"Patch":85 },
   "instanceNameFormat": "Echo trivy $(version)",
   
   "groups": [
@@ -187,7 +187,10 @@
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "The Aqua API Key is used to upload scan results to your Aqua Security account.",
-      "groupName": "commercial"
+      "groupName": "commercial",
+      "properties": {
+        "isSecret": true
+      }
     },
     {
       "name": "aquaSecret",
@@ -196,7 +199,10 @@
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "The Aqua API Secret is used to upload scan results to your Aqua Security account.",
-      "groupName": "commercial"
+      "groupName": "commercial",
+      "properties": {
+        "isSecret": true
+      }
     },
     {
       "name": "cosignOutput",

--- a/trivy-task/task.json.tmpl
+++ b/trivy-task/task.json.tmpl
@@ -187,7 +187,10 @@
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "The Aqua API Key is used to upload scan results to your Aqua Security account.",
-      "groupName": "commercial"
+      "groupName": "commercial",
+      "properties": {
+        "isSecret": true
+      }
     },
     {
       "name": "aquaSecret",
@@ -196,7 +199,10 @@
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "The Aqua API Secret is used to upload scan results to your Aqua Security account.",
-      "groupName": "commercial"
+      "groupName": "commercial",
+      "properties": {
+        "isSecret": true
+      }
     },
     {
       "name": "cosignOutput",

--- a/vss-extension.dev.json
+++ b/vss-extension.dev.json
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "trivy-official-dev",
   "publisher": "AquaSecurityOfficial",
-  "version": "1.11.81",
+  "version": "1.11.85",
   "name": "trivy-dev",
   "description": "Trivy is the world’s most popular open source vulnerability and misconfiguration scanner. It is reliable, fast, extremely easy to use, and it works wherever you need it.",
   "repository": {

